### DISCUSSION TITLE
Authorization on stratum server failed: Unsupported request eth_submitLogin

### DISCUSSION THREAD

**Discussion by jadeden1999** [coinfoundry/miningcore#975] [Q&A] [answered]
Authorization on stratum server failed: Unsupported request eth_submitLogin
gminer 
and same error on trex 
what could be the issue?

**Comment by Konstantin35** [comment, marked as answer]
> 
> 
> Authorization on stratum server failed: Unsupported request eth_submitLogin gminer and same error on trex what could be the issue?

https://github.com/coinfoundry/miningcore#ethereum

**Comment by jadeden1999** [comment]
t-rex.exe -a ethash -o stratum2+tcp://<yourpoolurl>:<port> -u 0x3e7fb699fdf5d47cfc1e3a1a939d5156006481d4
stratum2 protocol for t-rex and others 
works like a charm

**Reply by oliverw** [reply]
> t-rex.exe -a ethash -o stratum2+tcp://: -u 0x3e7fb699fdf5d47cfc1e3a1a939d5156006481d4 stratum2 protocol for t-rex and others works like a charm

Thanks for the update @jadeden1999
